matrix_content_scanner/config.py

```diff
@@ -120,3 +129,29 @@ def __init__(self, config_dict: Dict[str, Any]):
         self.scan = ScanConfig(**(config_dict.get("scan") or {}))
         self.crypto = CryptoConfig(**(config_dict.get("crypto") or {}))
         self.download = DownloadConfig(**(config_dict.get("download") or {}))
+        self.result_cache = ResultCacheConfig(**(config_dict.get("result_cache") or {}))
+
+
+def parse_user_value(
+    v: Optional[Union[str, float]], parser: Callable[[str], float]
+) -> Optional[float]:
```

Comment:
I considered that but figured it would be quite boilerplatey and more copy-pasting from Synapse than I was feeling comfortable with. But happy to do it if you feel like it'd be worth it.

Reply:
I'm not suggesting you copy-paste from Synapse - how you implement `parse_size` is up to you (calling out to `human_readable.parse_size` seems like a good solution).

But, as a caller of this stuff, `parse_size(foo)` is way easier to write and understand than `parse_user_value(foo, parse_size)`.

Reply:
Oh right, yes sorry I misunderstood you.